spec/features/idv/analytics_spec.rb

```diff
@@ -57,7 +57,7 @@
       verified_attributes: [],
       state: 'MT',
       state_id_jurisdiction: 'ND',
-      state_id_number: '#############',
+      id_number: '#############',
```

Comment:
I think this is what comes back from the proofing vendor, so we don't want to rename it.

Although the test fails so there's maybe another piece missing.

Reply:
I'm so glad you're as excited as I am, this was also bugging me so much!

As for the test failures, I think I just tracked this down... I think the various analytics failures are showing me that there is some shared code (app/controllers/concerns/idv/verify_info_concern.rb) using "state_id_number" which is what I feared. I'm starting to feel once again that renaming the field everywhere is a bit overengineered, because it wouldn't truly be everywhere - only in the in-person code, leaving us to manage two different variables in any shared code.

I'm considering reverting back to the "translate closer to the form" solution so as not to sink any more time into this. Curious your thoughts!

Reply:
After some more discussion and thought I went ahead and reset to the previous solution which passes specs and doesn't rename these fields.